lombok.config

```diff
@@ -0,0 +1 @@
+lombok.anyConstructor.addConstructorProperties=true
```

Comment:
I'd prefer to have the annotations and constructor than an extra config file.

Reply:
adding those constructors might be very error prone because it should depend on json properties - so when you add new or remove json property you also need update constructor and keep in in sync